app/src/main/java/co/develhope/chooseyourownbeer/network/BeersProvider.kt

```diff
@@ -4,11 +4,11 @@ import co.develhope.chooseyourownbeer.ui.model.BeerUi
 import co.develhope.chooseyourownbeer.network.dto.toListOfBeerUi
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
+import java.net.URL
 
-
+const val URL="https://api.punkapi.com/v2/"
 class BeersProvider {
     private var retrofit = Retrofit.Builder()
-        .baseUrl("https://api.punkapi.com/v2/")
```

Comment:
Non devi cancellare .baseUrl, in questo modo retrofit non riesce a raggiungere l'url per l'API. Hai provato a compilare l'app?

Reply:
ok che l'ha spostata ma devi cmq settare il base url altrimenti Retrofit come fa a asapere dove pescare le cose?